src/internal/tabbable.ts

```diff
@@ -1,4 +1,4 @@
-// Determines if the specified element is tabbable using heuristics inspired by https://github.com/focus-trap/tabbable
+/** Determines if the specified element is tabbable using heuristics inspired by https://github.com/focus-trap/tabbable */
```

Comment:
Do we care about JSDoc comments for things that aren't exported?

Reply:
Does it hurt? At the very least you get the IDE tooltip with the explanation as a developer of this package.

I think that if a comment is useful to the author of a function, then it should just be a normal comment, but if the comment is useful to the user of a function, then it should be a JSDoc comment so the user elsewhere in the codebase can easily see it. These comments seem like they are useful for the user of the function.

But let me ask you, do you care about JSDoc comments for internal methods? Up to you!

Reply:
I guess the tips are nice even if I don't use them for internal methods (most utils are pretty small anyways). I'll try to get into the habit of doing this.